config/tls_termination.go

```diff
@@ -27,6 +27,7 @@ func (tt tlsTermination) ApplyTLS(cfg *tlsOptions) {
 		// TODO: implement this in the tunnel `Accept` call.
 		panic("automatic tls termination in-app is not yet supported")
 	case TLSAtEdge:
+		cfg.terminateAtEdge = true
 		cfg.KeyPEM = tt.key
 		cfg.CertPEM = tt.cert
```

Comment:
One alternative would be to make sure that these are always non-nil here, but I think the extra boolean makes the intent clearer :+1:

Reply:
It does look like I could set zero-length arrays and get the same server effect, if you'd rather I went that route.